views/home.jsx

```diff
@@ -0,0 +1,32 @@
+var React = require('react');
+class Home extends React.Component {
+  render() {
+    console.log("////");
+    console.log("Render is working !!");
+    console.log("////");
+    console.log(this.props.id);
+    console.log("This is the object in home" + this.props.recipe);
+
+    let recipeArray = this.props.recipe
+    console.log(recipeArray[this.props.id]);
+    const checkID = id => {
+        console.log("checking ID")
+        return id === this.props.recipe.id
+    }
+
+    const correctRecipe = recipeArray.filter(checkID);
+
```

Comment:
Added some code in to print out the desired content, tried to use filter but can't get it to work

[https://floobits.com/kokchuan/SEI22codezz/file/recipe-keeper/views/home.jsx:1](url)